tests/unit/views/societies/AllSocieties.spec.js

```diff
@@ -0,0 +1,114 @@
+import { expect } from 'chai';
+
+import SocietyTile from '@/components/society/SocietyTile';
+import InfiniteScroll from '@/components/ui/InfiniteScroll';
+import AllSocieties from '@/views/societies/AllSocieties';
+
+import {
+  executeWithServer,
+  generateMountOptions,
+  mountWithRouterMock,
+  waitFor,
+} from '../../helpers';
+
+describe('All Societies', () => {
+  let allSocietiesComponent, server;
+  beforeAll(async () => {
+    server = await executeWithServer(() => {}, false);
+  });
+  afterAll(() => {
+    server.shutdown();
+  });
+  beforeEach(async () => {
+    allSocietiesComponent = await mountWithRouterMock(
+      AllSocieties,
+      generateMountOptions(['apollo'])
+    );
+  });
+
+  it('contains an infinite scroll instance', () => {
+    expect(allSocietiesComponent.findComponent(InfiniteScroll).exists()).to.be
+      .true;
+  });
+
+  describe('with no productions', () => {
+    it('displays no productions notice', async () => {
+      await waitFor(
+        () => !allSocietiesComponent.findComponent(InfiniteScroll).vm.loading
+      );
+      expect(allSocietiesComponent.text()).to.contain('Loading Societies');
```

Comment:
This test doesn't make sense... Look at the name of the test and this expectation